packages/manager/modules/iam/src/components/onboarding/onboarding.controller.js

```diff
@@ -0,0 +1,24 @@
+import { ROUTES } from '@iam/routes';
+
+export default class IamOnboardingController {
+  /* @ngInject */
+  constructor($state, $translate) {
+    this.$state = $state;
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    this.guides = this.onboardingGuides.map((guide) => {
+      const { key, text, url } = guide;
+      return {
+        title: text,
+        description: this.$translate.instant(
+          `iam_onboarding_guide_${key}_description`,
+        ),
+        link: url,
+      };
+    });
```

Comment:
I see 2 options to avoid having this map
- Double the keys insides GuideService
- Modify the keys where the guides are used

Reply:
It's applied.
I changed the key for use the same in onboarding and in header.